app/http.service.ts

```diff
@@ -0,0 +1,15 @@
+import {Injectable} from '@angular/core';
+import {Http, Response} from '@angular/http';
+import 'rxjs/Rx';
+
+@Injectable()
+export class HttpService {
```

Comment:
this service doesn't look like a core service that should be stored in a root folder
can it be renamed and moved to an appropriate component's folder?